pkg/utils/eden.go

```diff
@@ -152,21 +157,73 @@ func StatusAdam() (status string, err error) {
 	return state, nil
 }
 
-//StartEServer function run eserver to serve images
-func StartEServer(commandPath string, serverPort int, imageDist string, logFile string, pidFile string) (err error) {
-	commandArgsString := fmt.Sprintf("server -p %d -d %s -v %s", serverPort, imageDist, log.GetLevel())
-	log.Infof("StartEServer run: %s %s", commandPath, commandArgsString)
-	return RunCommandNohup(commandPath, logFile, pidFile, strings.Fields(commandArgsString)...)
+//StartEServer function run eserver in docker
+//if eserverForce is set, it recreates container
+func StartEServer(serverPort int, imageDist string, eserverForce bool, eserverTag string) (err error) {
+	portMap := map[string]string{"8888": strconv.Itoa(serverPort)}
+	volumeMap := map[string]string{"/eserver/run/eserver/": fmt.Sprintf("%s", imageDist)}
```

Comment:
We need to make sure that the volume map is optional. IOW, it should be possible to start  eserver in the cloud with a completely empty state and populate it only via http interactions.

Reply:
We are moving in that direction. In the next PR, support for uploading files will be added and the transition will be completed.